sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroUtils.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.beam.sdk.io;
-
-import java.io.Serializable;
-import org.apache.avro.Schema;
-import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Function;
-import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Supplier;
-import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Suppliers;
-
-/** Helpers for working with Avro. */
-class AvroUtils {
```

Comment:
There is one more similar class in `AvroCoder`, but I guess changing it will introduce incompatible change into `AvroCoder` and break streaming pipelines, so let's keep it for now.

Reply:
Ok just for curiosity which class is it?

Reply:
`AvroCoder.SerializableSchemaSupplier`

Reply:
But we can't change it, or it will break Java serialization.

Reply:
Actually, I don't know how it works now, but it uses classes from vendored guava, that changes namespace each time we change guava version. I'm wondering if we already broke it without noticing.

Reply:
I asked on dev, but probably we already broke it a couple of times